tools/cldr-code/src/main/java/org/unicode/cldr/tool/ToolConstants.java

```diff
@@ -40,7 +40,7 @@ public enum ChartStatus {
                             .map(x -> VersionInfo.getInstance(x))
                             .collect(Collectors.toList()));
 
-    public static final String DEV_VERSION = "43";
+    public static final String DEV_VERSION = "44";
```

Comment:
oops

Reply:
I needed to generate the charts ... and that required bumping the version.